core/src/test/java/com/netease/arctic/CatalogMetaTestUtil.java

```diff
@@ -62,6 +62,37 @@ public static CatalogMeta createArcticCatalog(File arcticBaseDir) throws IOExcep
     return meta;
   }
 
+  public static CatalogMeta createHadoopCatalog(File arcticBaseDir) throws IOException {
```

Comment:
As the module core use `CatalogTestHelpers#buildCatalogMeta` to build the catalog meta, would you like to move this class to the spark common module?

Or we should try to unity the catalog meta construct util across modules.

Reply:
This method and class are removed, using CatalogTestHelper instead.